requirements.txt

```diff
@@ -1,3 +1,17 @@
+attrs==21.4.0
```

Comment:
was missing packages while running pytest. Exported 'pip freeze' again

Reply:
Can we make a test-requirements.txt?